gnomad/utils/annotations.py

```diff
@@ -334,6 +334,9 @@ def annotate_freq(
     pop_expr: Optional[hl.expr.StringExpression] = None,
     subpop_expr: Optional[hl.expr.StringExpression] = None,
     additional_strata_expr: Optional[Dict[str, hl.expr.StringExpression]] = None,
+    additional_strata_grouping_expr: Optional[
+        Dict[str, hl.expr.StringExpression]
+    ] = None,
     downsamplings: Optional[List[int]] = None,
```

Comment:
Will this break any of our other code that assumes ordering?

Reply:
I checked gnomad_methods and gnomad_qc. gnomad_methods does not called annotate_freq. gnomad_qc/v3 does but only assumes ordering of `additional_strata_expr` when passing subsets, so this change will not impact that code. All other calls use keyword assignment.